### ISSUE TITLE
404 resource not found in iis6

### ISSUE THREAD

**Issue by GomesNayagam** [domaindrivendev/Swashbuckle#154]
Hi the /swagger/ui/index.html is working fine in local development box whereas it throws 404 at sever. the only difference is local is iis7 and dev is iis6

**Comment by domaindrivendev** [comment]
Hi there

Can you let me know which version of SB you are currently using?

To serve the swagger-ui using only the WebApi pipeline, the SB handlers need to be called for URL's with extensions (.html, .js, .css etc.). IIS disables this by default and handles these requests with a static file handler, bypassing the WebApi pipeline. This is why you're seeing a 404 because the static file handler is just looking for the files on your file system.

In summary, SB needs to opt out of this behavior for the swagger-ui requests and has gone through a couple of iterations in it's method for doing this. Can you check your web.config, and show me how your modules and handlers (under system.webServer) are configured. I think a slight tweak here may solve your problem.

Thanks

**Comment by GomesNayagam** [comment]
Anyway we moved to IIS7 and issue resolved but others may have similar issue. I was using SB 1.0. Below is the config details 
 <system.webServer>
    <modules runAllManagedModulesForAllRequests="true" />
    <validation validateIntegratedModeConfiguration="false" />
  <handlers>
      <add name="SwashbuckleUiHandler" verb="*" path="swagger/ui/*" type="Swashbuckle.Application.SwaggerUiHandler, Swashbuckle.Core" preCondition="managedHandler" />
    </handlers></system.webServer>
